certified-connectors/Webhood URL Scan/apiDefinition.swagger.json

```diff
@@ -0,0 +1,372 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Webhood - Beta API",
+    "description": "This is the Swagger 2.0 schema for the Webhood URL scanner API",
+    "contact": {
+      "name": "Webhood",
+      "url": "https://webhood.io",
+      "email": "contact@webhood.io"
+    },
+    "version": "1.0.0"
+  },
+  "host": "demo.webhood.io",
+  "basePath": "/api",
+  "externalDocs": {
+    "description": "Find out more about Webhood URL Scanner",
+    "url": "https://webhood.io"
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.webhood.io"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.webhood.io/docs/about/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Security;IT Operations"
+    }
+  ],
+  "tags": [
+    {
+      "name": "scanner",
+      "description": "Operations about scanner"
+    }
+  ],
+  "schemes": [
+    "https",
+    "http"
+  ],
+  "paths": {
+    "/beta/scans": {
+      "post": {
+        "security": [{"apiKey": []}],
+        "summary": "Create a new scan",
+        "description": "Create a new scan to be initiated by the scanner",
+        "operationId": "initScan",
+        "tags": [
+          "scanner"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Create a new scan to be initiated by the scanner",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ScanRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully initiated scan",
+            "schema": {
+              "$ref": "#/definitions/Scan"
+            }
+          },
+          "202": {
+            "description": "Successfully initiated scan asynchronously",
+            "schema": {
+              "$ref": "#/definitions/Scan"
+            }
+          },
+          "400": {
+            "description": "Something went wrong while processing your request"
+          },
+          "401": {
+            "description": "The request requires valid record authorization token to be set"
+          },
+          "403": {
+            "description": "Authentication or authorization failure"
+          }
+        }
+      },
+      "get": {
+        "security": [{"apiKey": []}],
+        "tags": [
+          "scanner"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Get past scans, optionally filter by status",
```

Comment:
The Description and x-ms-summary properties are required for get.  Please refer the schema and the documentation below. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/apiDefinition.swagger.schema.json

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#checklist-before-submitting

Please find the errors below during validation. 

##[error]The 'description' property is required. ValuePath: /paths/~1beta~1scans/get

##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1beta~1scans~1{scanId}/get/parameters/0